docs/source/gbd2017_models/causes/neonatal_disorders/index.rst

```diff
@@ -0,0 +1,265 @@
+.. _2017_cause_neonatal_disorders:
+
+==================
+Neonatal Disorders
+==================
+
+.. todo::
+
+  Add a brief introductory paragraph for this document.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Disease Overview
+----------------
+
+.. todo::
+
+   Add a general clinical overview of the cause.
+
+
+GBD 2017 Modeling Strategy
+--------------------------
+
+.. todo::
+
+  Add an overview of the GBD modeling section, citing
+  [GBD-2017-YLD-Appendix-Neonatal-Disorders]_ and
+  [GBD-2017-CoD-Appendix-Neonatal-Disorders]_.
+
+
+Cause Hierarchy
++++++++++++++++
+
+Restrictions
+++++++++++++
+
+The following table describes any restrictions in GBD 2017 on the effects of
+this cause (such as being only fatal or only nonfatal), as well as restrictions
+on the ages and sexes to which the cause applies.
+
+.. list-table:: GBD 2017 Cause Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
```

Comment:
@kiranklc suggested that we could add a note in this table to clarify that this cause is NOT YLL only in GBD, but that we will be treating it as such for our cause model. Might be nice to highlight this here for clarity.

Reply:
Good point, thanks @alibow and @kiranklc